app/helpers/application_helper.rb

```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+	# Returns the full title on a per-page basis.
+  def full_title(page_title = '')
```

Comment:
```suggestion
  def full_title page_title = ""
```
trong rails dùng " thay cho ' nhé em

Reply:
ready